Readme.md

```diff
@@ -1,105 +1,91 @@
-* **[Neos.EventSourcing.MonitoringHelper](https://github.com/neos/Neos.EventSourcing.MonitoringHelper)**: aspect to monitor performance of the ```Neos.EventSourcing``` infrastructure
-* **[Neos.EventStore](https://github.com/neos/Neos.EventStore)**: event store to support event sourcing
-* **[Neos.EventSourcing.InMemoryStorageAdapter](https://github.com/neos/Neos.EventSourcing.InMemoryStorageAdapter)**: in memory event storage, mainly for testing
-* **[Neos.EventSourcing.DatabaseStorageAdapter](https://github.com/neos/Neos.EventSourcing.DatabaseStorageAdapter)**: doctrine dbal based event storage
-
-More storage can be added later (Redis, ...).
-
-# Folder Structure
-
-This is a PSR-4 package structure:
-
-    Your.Package/
-      Application/
+    Your.BoundedContextPackage.Command/
+      Classes/
         Controller/
-        Command/
+          [UseCase]Controller.php
+          …
         Service/
-        Projection/
-            [ProjectionName]/
-                [ProjectionName]Finder.php
-                [ProjectionName]Projector.php
-                [ProjectionName].php
-
-      CommandHandler/
-
-      Domain/
-        Service/
-        Aggregate/
+          [SomeDomain]Service.php
+          …
+        Model/
           [YourAggregate]/
             Command/
+              [DoSomething].php
+              …
             Event/
-            Service/
-            YourAggregate.php
-            YourAggregateRepository.php
+              [SomethingHappened].php
+              …
+            [YourAggregate].php
+            [YourAggregate]CommandHandler.php
+            [YourAggregate]Repository.php
+          …
+        Process/
+          [Some]ProcessManager.php
+          …
+
+    Your.BoundedContextPackage.Query/
+      Classes/
+        Controller/
+          [ReadUseCase]Controller.php
+          …
+        Dto/
+          [SomeQueryDto].php
+          …
+        Service/
+          [SomeNonDomain]Service.php
+          …
+        Projection/
+          [ProjectionName]/
+            [ProjectionName].php
+            [ProjectionName]Finder.php
+            [ProjectionName]Projector.php
+          …
+
+The reason to separate your Command and Query sides into own Packages is to make it clear those are fully separate
+parts of your application. However, it is no problem to have both parts inside a single package.
+
+Also note that this is just a recommendation and not a necessary structure for anything to work as expected.
+The only requirements are the naming conventions for the Aggregate Repository and the Projection Finder and Projector.
+Those can be easily overruled in the code through the respective properties ```aggregateClassName``` and ```readModelClassName``` though.
 
-      EventListener/
+# Command / Write side
 
-# Components
+Your write side models your core domain logic, ensuring consistency and keeping business rules intact.
```

Comment:
Yes, and those are the parts where you should apply DDD and ES on :) But I get your point